spec/helpers/barnardos/form_helpers_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
+RSpec.describe Barnardos::ActionView::FormHelpers, :type => :helper do
+  include RSpecHtmlMatchers
+
+  describe '#labelled_text_field_tag' do
+    ##
+    # <!-- Example HTML output -->
+    # <div class="textfield " id="participant_name-wrapper">
+    #   <label class="textfield__label textfield__label--bold" for="participant_name">
+    #     What is the name of the research participant?
+    #     <span class="textfield__hint">Full name</span>
+    #   </label>
+    #   <input id="participant_name" class="textfield__input" name="participant_name" type="text">
+    # </div>
+    context 'with a name and a label parameter' do
+      let(:name) { :participant_name }
+      let(:label) { 'What is the name of the research participant?' }
+
+      subject(:rendered) { helper.labelled_text_field_tag(name, label) }
+
+      it 'outputs an wrapper div with the class and id' do
+        expect(rendered).to have_tag('div', with: { class: 'textfield', id: 'participant_name-wrapper' })
+      end
+
+      it 'renders a classed bold label and input as sibling children of the div' do
```

Comment:
Can we test not only that it has the elements but also has the correct structure and order?

We'll need to add more tests in time to confirm things like picking up optional params like hint and error